pysoa/server/server.py

```diff
@@ -107,8 +108,9 @@ def __init__(self, settings):
             raise AttributeError('Server subclass must set service_name')
 
         self.async_event_loop = None
-        if asyncio:
-            self.async_event_loop = asyncio.get_event_loop()
+        if has_asyncio:
+            self._async_event_loop_thread = AsyncEventLoopThread()
+            self.async_event_loop = self._async_event_loop_thread.loop
```

Comment:
A couple things:
- You can make `async_event_loop` private (`_async_event_loop`)
- We're currently adding the `async_event_loop` to the job request (https://github.com/eventbrite/pysoa/blob/e195507f3673903744abd5675018324b198c3aad/pysoa/server/server.py#L313) and to the action request (https://github.com/eventbrite/pysoa/blob/e195507f3673903744abd5675018324b198c3aad/pysoa/server/server.py#L413). Since we want everybody to use `AsyncEventLoopThread.run_coroutine`, we should transition away from adding the `async_event_loop` attribute to the job/action requests and instead add an `run_async_coroutine` (a lambda that calls `AsyncEventLoopThread.run_coroutine`, I don't want to expose the whole thread where someone could call `run` or `join`). We might have to have them both at the same time as a transition period.